modules/ding_ting_frontend/ding_ting_frontend.info

```diff
@@ -25,7 +25,6 @@ dependencies[] = ting_proxy
 dependencies[] = ting_reference
 dependencies[] = ting_relation
 dependencies[] = ting_search
-dependencies[] = ting_search_autocomplete
```

Comment:
:+1: - but remember that an autocomplete module should still be enabled by default on new installations. Consider enabling `ting_search_autocomplete` as a part of the profile installation.